src/comp/Avater.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react"
+import "./Avater.css"
+
+import AvaterToolTip from "./AvaterToolTip"
+
+import 'react-tippy/dist/tippy.css'
+
+import {
+  Tooltip,
+} from 'react-tippy';
+
+type AvaterProps = {
+	uuid: string
+	hiddenName: boolean
+}
+
+type AvaterState = {
+	name: string
+	picurl: string
+	is_loaded: boolean
+}
+
+class Avater extends React.Component<AvaterProps, AvaterState> {
+	
+	static defaultProps = {
+    hiddenName: false,
+  };	
```

Comment:
not used variable
```suggestion

```

Reply:
this is required for omitting a prop `hiddenName`.
```tsx
<Avater uuid="something" />            /* hiddenName is false */
<Avater uuid="something" hiddenName /> /* hiddenName is true */
```

Reply:
oh sorry didn't notice